modules/ROOT/pages/idt.adoc

```diff
@@ -0,0 +1,200 @@
+= Interrupt Descriptor Table
+:description: 
+:keywords: interrupts, x86
+:page-category: interrupts
+:source-language: c
+:table-caption!:
+
+== Overview
+
+The *Interrupt Descriptor Table* is a binary data structure specific to xref:x86.adoc[x86]'s 32-bit (IA-32) and 64-bit (x86-64) variants. It is used in xref:x86.adoc#_protected_mode_32_bit[*Protected Mode*] and xref:x86.adoc#_long_mode_x86_64_only[*Long Mode*] and a similar *Interrupt Vector Table* exists in xref:x86.adoc#_real_mode[*Real Mode*].
+
+The *IDT* is responsible for informing the CPU of the location and operation modes of *Interrupt Service Routines* corresponding to one of 256 *Interrupt Vectors*.
```

Comment:
IDT must not be in bold more than once. This applies to all other terms as well.